File: programme/average-of-numbers/average_of_numbers .cpp
// Write a programmeto find average of numbers
#include <iostream>

using namespace std;

int main()
{
    int N, num;
    cout << "Numbers count : ";
    cin >> N;
    cout << "\nEnter " << N << " Numbers" << endl;
    int sum = 0;
    for (int i = 0; i < N; i++)
    {
        cout << "Enter : ";
        cin >> num;
        sum += num;
    }
    cout << "\nAverage is " << (sum * 1.0) / N << endl;
    ;
}

File: programme/average-of-numbers/average-of-numbers.go
package main

import "fmt"

func main() {
	var n int
	var sum float64
	fmt.Print("Enter the number of numbers : ")
	fmt.Scan(&n)
	for i := 0; i < n; i++ {
		var num float64
		fmt.Print("Enter the number : ")
		fmt.Scan(&num)
		sum += num
	}
	fmt.Println("\nThe average of the numbers is :", sum/float64(n))
}
